src/js/media-time-range.js

```diff
@@ -313,77 +327,84 @@ class MediaTimeRange extends MediaChromeRange {
     style.transform = `translateX(${boxPos}px)`;
   }
 
-  _enableBoxes() {
```

Comment:
It's hard to follow the diff after this point. Was this a simple moving around of code or was there a refactor happening?

Reply:
yes, this mostly moves the handlers out of `_enableBoxes()` into private properties so that a `_disableBoxes()` method could be created.